Source/Commands.c

```diff
@@ -211,7 +207,7 @@ static void loginCommand(Server* server, char command[30], char* message, uint8
             for (uint8 j = 0; j < levelLen; ++j) {
                 level[j] = tolower(level[j]);
             }
-            uint failed = 0;
+            unsigned int failed = 0;
```

Comment:
There is no need to waste 32bits here. This was a typo and was meant to be uint8.

Can you replace it with uin8 ?

Thank you